installing/installing_ibm_power/preparing-to-install-on-ibm-power-vs.adoc

```diff
@@ -0,0 +1,58 @@
+include::snippets/technology-preview.adoc[]
+
+[id="requirements-for-installing-ocp-on-ibm-power-vs"]
+== Requirements for installing {product-title} on {ibmpowerProductName} Virtual Server
+
+Before installing {product-title} on {ibmpowerProductName} Virtual Server, you must create a service account and configure an IBM Cloud account. See xref:../../installing/installing_ibm_power/installing-ibm-cloud-account-power-vs.adoc#installing-ibm-cloud-account-power-vs[Configuring an IBM Cloud account for {ibmpowerProductName} Virtual Server] for details about creating an account, configuring DNS and supported {ibmpowerProductName} Virtual Server regions.
+
+You must manually manage your cloud credentials when installing a cluster to {ibmpowerProductName} Virtual Server. Do this by configuring the Cloud Credential Operator (CCO) for manual mode before you install the cluster.
+
+[id="choosing-a-method-to-install-ocp-on-ibm-power-vs"]
+== Choosing a method to install {product-title} on {ibmpowerProductName} Virtual Server
+
+You can install {product-title} on {ibmpowerProductName} Virtual Server using installer-provisioned infrastructure. This process involves using an installation program to provision the underlying infrastructure for your cluster. Installing {product-title} on {ibmpowerProductName} Virtual Server using user-provisioned infrastructure is not supported at this time.
+
+See xref:../../architecture/architecture-installation.adoc#installation-process_architecture-installation[Installation process] for more information about installer-provisioned installation processes.
+
+[id="choosing-an-method-to-install-ocp-on-power-vs-installer-provisioned"]
+=== Installing a cluster on installer-provisioned infrastructure
+
+You can install a cluster on {ibmpowerProductName} Virtual Server infrastructure that is provisioned by the {product-title} installation program by using one of the following methods:
+
+* **xref:../../installing/installing_ibm_power/installing-ibm-power-vs-customizations.adoc#installing-ibm-power-vs-customizations[Installing a customized cluster on {ibmpowerProductName} Virtual Server]**: You can install a customized cluster on {ibmpowerProductName} Virtual Server infrastructure that the installation program provisions. The installation program allows for some customization to be applied at the installation stage. Many other customization options are available xref:../../post_installation_configuration/cluster-tasks.adoc#post-install-cluster-tasks[post-installation].
+
+* **xref:../../installing/installing_ibm_power/installing-ibm-powervs-vpc.adoc#installing-ibm-powervs-vpc[Installing a cluster on {ibmpowerProductName} Virtual Server into an existing VPC]**: You can install {product-title} on an existing {ibmpowerProductName} Virtual Server (VPC). You can use this installation method if you have constraints set by the guidelines of your company, such as limits when creating new accounts or infrastructure.
```

Comment:
Can we write it as : 
existing {ibmpowerProductName} Virtual Server Virtual Private Cloud (VPC) ?

@stevsmit could you please confirm ? Thanks.

Reply:
@alishaIBM I apologize. Please disregard this suggestion. While normally I would say that we should define abbreviations on first/new usages, this would deter from readability. That said, I do have a question: are Virtual Server and VPC the same thing? It seems odd to use the VPC abbreviation for Virtual Server, or to use it after saying ". . . Virtual Server". Could we maybe say

**You can install an {product-title} cluster on an existing Virtual Private Cloud (VPC) on {ibmpowerProductName}.** 

^ I'm not sure if that captures the exact idea. 

Alternatively, we could maybe remove the (VPC) abbreviation, eg: 

**You can install {product-title} on an existing {ibmpowerProductName} Virtual Server.**

Alternatively, we could use the exact language from the module: 

**You can Installing a cluster on IBM Power Virtual Server into an existing Virtual Private Cloud (VPC).**

Reply:
Sounds good to me. Thanks!